src/v3/src/util/rendererUtils.ts

```diff
@@ -26,10 +26,18 @@ export const isInputTextFieldElement = ({
     inputMeta: {
       type, options, name, secret,
     } = {} as Input,
-  },
+  } = {} as FieldElement['options'],
 }: FieldElement): boolean => (
-  ((type === 'string' || defaultType === 'string') && !options && !secret)
-      || (name === 'credentials.passcode' && !secret)
+  (((type === 'string' || defaultType === 'string') && !options && !secret)
+      || (name === 'credentials.passcode' && !secret))
+);
+
+export const isInputTextWithNoValidationFieldElement = (field: FieldElement): boolean => (
+  isInputTextFieldElement(field) && !field.options.inputMeta?.validate
+);
+
+export const isInputTextWithValidationFieldElement = (field: FieldElement): boolean => (
```

Comment:
Nit: do we need the `isInputTextWithNoValidationFieldElement` method when we have this one?  Could we just use this method and check for false value in `renderers.tsx`?

Reply:
they are both used in `src/v3/src/components/Form/renderers.tsx`